PowerUp/app/src/main/java/powerup/systers/com/data/entities/Scenario.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * @desc this class holds getter and setter methods for each dialogue scenario's status
+ * examples include setAsker() and getNextScenarioID()
+ */
+
+package powerup.systers.com.data.entities;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+
+@Entity(tableName = "Scenario")
+public class Scenario {
+
+    @PrimaryKey
+    private int scenarioId;
+
+    private String scenarioName;
+
+    private String timestamp;
+
+    private String asker;
+
+    private int avatar;
+
+    private int firstQuestionID;
+
+    private int completed;  // If equal to 1, the scenario is already completed
+
+    private int nextScenarioID;
+
+    private int replayed;   // If equal to 0, the scenario can be replayed
+
```

Comment:
I think adding blank line after each and every declaration here is too much. What do you think @m-murad @geekanamika ?

Reply:
Personally, I prefer to have space between variable declaration(only in model classes). It looks clean to me with spaces. I can remove it if Mentors don't agree it :)

Reply:
@niksj Can you please give your opinion on it? I'll update it's needed.